### PR TITLE
docs(datadog) document config.tag_api_name

### PR DESCRIPTION
**Pull request by kbacha**
Adds documentation related to changes in https://github.com/Kong/kong/pull/3037

image

### CODE REVIEW

File: app/plugins/datadog.md [hunk 1]
```diff
@@ -44,6 +44,7 @@ parameter                      | default     | description
 `config.port`<br>*optional*    | `8125`      | The port to send data to on the upstream server.
 `config.metrics`<br>*optional* | All metrics<br>are logged | List of Metrics to be logged. Available values are described at [Metrics](#metrics).
 `config.prefix`<br>*optional* | `kong` | String to be attached as prefix to metric's name.
+`config.tag_api_name`<br>*optional* | `false` | Whether to tag every metric with the api's name. If set to true then a tag of `api_name:<api-name>` will be included with metrics sent to datadog.
```
Comment (shashiranjan84) [inline app/plugins/datadog.md]: I think you should also mention that if `config.tag_api_name` is `true`, API name will not be prefixed in metrics identifier.

Comment (kbacha) [reply app/plugins/datadog.md]: Absolutely. I've updated it.

**Comment by hbagdi** [comment]
@kbacha seems like there is a conflict due to the changes introduced in `0.13.0`.

**Comment by kbacha** [comment]
@hbagdi Resolved!

**Comment by coopr** [comment]
Thanks @kbacha  - we've just made Kong Hub https://docs.konghq.com/hub/ live, which has caused some changes to the files you edited in this PR. Please reapply your changes to the new files you will now find at https://github.com/Kong/docs.konghq.com/tree/master/app/_hub/kong-inc

**Comment by hbagdi** [comment]
Closing as it stalled.

**Pull request closed without merging**